unix/src/machine_stats/plugins/cpu_utilization.py

```diff
@@ -0,0 +1,27 @@
+def setup(app):
+    timeout = app.args.cpu_utilization_timeout
+    if timeout > 0:
```

Comment:
Cool, so if a person runs `machine-stats --cpu-utilization-timeout 0` then it will run without any of the CPU plugin code here, and otherwise it will run this plug and include CPU data in the results. Did I get that right?

Reply:
Yes, that's correct.

Reply:
Hey @przmv - if it is on _by default_, can we update the README here to reflect this?
https://github.com/tidalmigrations/machine_stats/blob/master/unix/README.md#data-captured